### PR TITLE
Some pi and controller functions

### PR DESCRIPTION
**Pull request by Mr-Wiseguy**
Changed __osPiRawStartDma to match the mdebug type info

### CODE REVIEW

File: src/io/pidma.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    mb->piHandle = NULL;
+    
+    if (priority == OS_MESG_PRI_HIGH) {
+        ret = osJamMesg(osPiGetCmdQueue(), (OSMesg)mb, OS_MESG_NOBLOCK);
+    } else {
+        ret = osSendMesg(osPiGetCmdQueue(), (OSMesg)mb, OS_MESG_NOBLOCK);
+    }
+    
+    return ret;
+}
```
Comment (ethteck) [inline src/io/pidma.c:30]: new line

File: src/io/pimgr.c [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+OSPiHandle __Dom1SpeedParam ALIGNED(8);
+OSPiHandle __Dom2SpeedParam ALIGNED(8);
+OSPiHandle *__osCurrentHandle[2] ALIGNED(8) = {&__Dom1SpeedParam, &__Dom2SpeedParam};
+static OSThread piThread;
+static char piThreadStack[OS_PIM_STACKSIZE];
+static OSMesgQueue piEventQueue;
+static OSMesg piEventBuf[1];
+
+void osCreatePiManager(OSPri pri, OSMesgQueue *cmdQ, OSMesg *cmdBuf, s32 cmdMsgCnt) {
+	u32 savedMask;
```
Comment (ethteck) [inline src/io/pimgr.c:16]: spaces instead of tabs

Comment (Mr-Wiseguy) [reply src/io/pimgr.c]: Fixed

File: src/io/pidma.c (hunk 1 above)
Comment (Mr-Wiseguy) [reply src/io/pidma.c]: Fixed

**Review by ethteck — APPROVED** [review]

**Pull request merged**